website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -1,8 +1,8 @@
-# Migrate to the EE Helm Chart
+# Migrate
 
-If you are using Appsmith Business Edition, with the Helm chart from `helm.appsmith.com`, this document will help you move to the new Helm chart from `helm-ee.appsmith.com`.
+Follow this guide to move from Appsmith Business Edition Helm chart installation(`helm.appsmith.com`) to Business Edition Helm chart(`helm-ee.appsmith.com`). The new `helm-ee.appsmith.com` installation supports Horizontal Pod Auto Scaling (HPA) which allows Appsmith pods to be automatically scaled based on current load.
 
-## Take backup
+## Export backup
```

Comment:
Are you sure of the wording here? We usually _export data_, or _take a backup_, but _exporting backup_ sounds a bit off to me. Either way is fine though, just wanted to point out.

Reply:
We are using 'Export data' everywhere in the docs for taking backups.

Reply:
Let's use the verb first